packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -265,7 +265,14 @@ const makeRenderer = (
 
 		// otherwise render using the API and React router
 		const addFlags = store => {
-			const memberObj = (store.getState().api.self || {}).value || {};
+			// use the api self object first if it exists,
+			// else, use the member id on the member cookie,
+			// else, this person's a guest and they'll get a
+			// default id of 0
+			const memberObj = (store.getState().api.self || {}).value || {
```

Comment:
ah, I remember this issue now and why we had to wait until the API calls had happened. Many feature flags are targeted at email addresses, so sending only the member ID won't work. We have a catch-22 here, where you need the full member object from the API in order to select the right flags, and you need to right flags in order to make the right API calls :/